presidio-analyzer/presidio_analyzer/recognizer_registry_provider.py

```diff
@@ -0,0 +1,156 @@
+import re
+import yaml
+import os
+import logging
+from pathlib import Path
+from typing import Optional, Union, List, Tuple, Dict, Any
+
+from presidio_analyzer import RecognizerRegistry, EntityRecognizer, PatternRecognizer
+from presidio_analyzer.nlp_engine import NlpEngineProvider, NlpEngine
+import presidio_analyzer.recognizer_registry
+
+logger = logging.getLogger("presidio-analyzer")
+
+
+class RecognizerRegistryProvider:
+    """
+    Utility class for loading Recognizer Registry.
+
+    Use this class to load recognzer registry from a yaml file
+    """
+
+    def __init__(self, conf_file: Optional[Union[Path, str]] = None, registry_configuration: Optional[Dict] = None):
+        self.configuration = self._get_configuration(conf_file, registry_configuration)
```

Comment:
nit: better be pass named parameters to avoid issues with renaming or changes to parameters in the future
```suggestion
        self.configuration = self._get_configuration(conf_file=conf_file, registry_configuration=registry_configuration)
```